.github/workflows/generated-files.yml

```diff
@@ -1,12 +1,12 @@
-name: build
+name: validate-generated-files
```

Comment:
Nitpick: Can we use the same name for the workflow as the filename please? 🙏 Makes it easier to find it when debugging the action.